### PR TITLE
Release 10.0.1, 9.1.1, 9.0.3, 8.5.5, and 8.4.5

### PR DESCRIPTION
**Pull request by Jafaral**

### CODE REVIEW

File: content/release/10.0.1.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+---
+Title: FRRouting Release 10.0.1
+Release: 10.0.1
+GitHub: https://github.com/FRRouting/frr/releases/tag/frr-10.0.1
+Date: 2024-06-06
+---
+
+FRRouting Release 10.0.1 Available for Download
+=============================================
```
Comment (ton31337) [inline content/release/10.0.1.md:9]: Could you align `=` with the title?

File: content/release/10.0.1.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+RPM Packages - https://rpm.frrouting.org
+
+Snaps - https://snapcraft.io/frr
+
+Docker - [quay.io/frrouting/frr:10.0.1](https://quay.io/repository/frrouting/frr/manifest/sha256:572c60d3d678887bbd97281e7995a976f7eceeb7f40bf43a13cbd7905d52af41)
+
+## Fixed CVEs
+- [CVE-2024-31950](https://frrouting.org/security/cve-2024-31950)
+- [CVE-2024-31951](https://frrouting.org/security/cve-2024-31951)
```
Comment (ton31337) [inline content/release/10.0.1.md:25]: Shouldn't we include also these: CVE-2024-34088, CVE-2024-31949, CVE-2024-31948, CVE-2024-27913?

File: content/release/10.0.1.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+* Fix `show route-map` command when calling via `do`
+* Show `ip ospf network ...` even if it's not the same as the interface type
+
+zebra
+* Deny the routes if ip protocol cli refers to an undefined rmap
+* Fix encoded dnssl length
+* Fix evpn svd based remote nh neigh del
+* Fix mpls command
+
+**Full Changelog**: https://github.com/FRRouting/frr/compare/frr-10.0.1...frr-10.0.1.1
```
Comment (ton31337) [inline content/release/10.0.1.md:112]: I think this should be `https://github.com/FRRouting/frr/compare/frr-10.0...frr-10.0.1`

File: content/release/8.4.5.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+- [CVE-2023-38802](https://frrouting.org/security/cve-2023-38802)
+- [CVE-2023-46752](https://frrouting.org/security/cve-2023-46752)
+- [CVE-2023-46753](https://frrouting.org/security/cve-2023-46753)
+- [CVE-2023-47235](https://frrouting.org/security/cve-2023-47235)
+- [CVE-2024-31948](https://frrouting.org/security/cve-2024-31948)
+
+## Bug Fixes
+
+babeld
+* Fix #11808 to avoid infinite loops
```
Comment (ton31337) [inline content/release/8.4.5.md:23]: This link leads to nowhere.

File: content/release/10.0.1.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+GitHub: https://github.com/FRRouting/frr/releases/tag/frr-10.0.1
+Date: 2024-06-06
+---
+
+FRRouting Release 10.0.1 Available for Download
+=============================================
+
+We are pleased to announce FRR release 10.0.1.
+
+FRR 10.0.1 brings a long list of enhancements and fixes with 938 commits from 54 developers. Thanks to all contributors.
```
Comment (ton31337) [inline content/release/10.0.1.md:13]: Is this an updated info?

File: content/release/10.0.1.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+
+Debian Packages - https://deb.frrouting.org
+
+RPM Packages - https://rpm.frrouting.org
+
+Snaps - https://snapcraft.io/frr
+
+Docker - [quay.io/frrouting/frr:10.0.1](https://quay.io/repository/frrouting/frr/manifest/sha256:572c60d3d678887bbd97281e7995a976f7eceeb7f40bf43a13cbd7905d52af41)
```
Comment (ton31337) [inline content/release/10.0.1.md:21]: The link needs to be updated after the images are built.

File: content/release/8.4.5.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+* Revert "cleanup memory allocations on shutdown"
+
+vtysh
+* Print uniq lines when parsing `no service ...`
+
+zebra
+* Deny the routes if ip protocol cli refers to an undefined rmap
+* Fix connected route deletion when multiple entry exists
+
+**Full Changelog**: https://github.com/FRRouting/frr/compare/frr-8.4.4...frr-4.4.5
```
Comment (ton31337) [inline content/release/8.4.5.md:93]: Wrong link.

File: content/release/8.5.5.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
+---
+Title: FRRouting Release 8.5.5
+Release: 8.5.5
+GitHub: https://github.com/FRRouting/frr/releases/tag/frr-8.5.5
+Date: 2024-06-06
+---
+
+FRRouting Release 8.5.5 Available for Download
+============================================
```
Comment (ton31337) [inline content/release/8.5.5.md:9]: Also, please align with the title.

File: content/release/9.0.3.md [hunk 9]
```diff
@@ -0,0 +1,125 @@
+---
+Title: FRRouting Release 9.0.3
+Release: 9.0.3
+GitHub: https://github.com/FRRouting/frr/releases/tag/frr-9.0.3
+Date: 2024-06-06
+---
+
+FRRouting Release 9.0.3 Available for Download
+============================================
```
Comment (ton31337) [inline content/release/9.0.3.md:9]: Ditto.

File: content/release/9.0.3.md [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+zebra
+* Add missing whitespace when printing route entry status
+* Deny the routes if ip protocol cli refers to an undefined rmap
+* Fix crash if macvlan link in another netns
+* Fix crash on macvlan link down/up
+* Fix nhg out of sync between zebra and kernel
+* The dplane_fpm_nl return path leaks memory
+
+
+**Full Changelog**: https://github.com/FRRouting/frr/compare/frr-9.0.3...frr-9.0.3
```
Comment (ton31337) [inline content/release/9.0.3.md:125]: Wrong link.

File: content/release/9.1.1.md [hunk 11]
```diff
@@ -0,0 +1,156 @@
+---
+Title: FRRouting Release 9.1.1
+Release: 9.1.1
+GitHub: https://github.com/FRRouting/frr/releases/tag/frr-9.1.1
+Date: 2024-06-06
+---
+
+FRRouting Release 9.1.1 Available for Download
+============================================
```
Comment (ton31337) [inline content/release/9.1.1.md:9]: Ditto.

File: content/release/9.1.1.md [hunk 12]
```diff
@@ -0,0 +1,156 @@
...
+* Don't deref vxlan-vni array
+* Fix crash if macvlan link in another netns
+* Fix crash on macvlan link down/up
+* Fix evpn svd based remote nh neigh del
+* Fix mpls command
+* Fix route deletion during zebra shutdown
+* The dplane_fpm_nl return path leaks memory
+
+
+**Full Changelog**: https://github.com/FRRouting/frr/compare/frr-9.1.1...frr-9.1.1.1
```
Comment (ton31337) [inline content/release/9.1.1.md:156]: Wrong link.

**Review by ton31337 — CHANGES_REQUESTED** [review]

File: content/release/8.4.5.md (hunk 4 above)
Comment (Jafaral) [reply content/release/8.4.5.md:23]: This isn't a link, just a PR number, whatever was there in the original PR comment

File: content/release/10.0.1.md (hunk 6 above)
Comment (ton31337) [reply content/release/10.0.1.md:19]: https://quay.io/repository/frrouting/frr/manifest/sha256:8ab9fe2ded0be732634bbd9a38e69c72ad450c7c9c41289b05a5ae7e26fdbe8b

File: content/release/9.1.1.md [hunk 13]
```diff
@@ -0,0 +1,154 @@
...
+
+We are pleased to announce FRR release 9.1.1.
+
+Debian Packages - https://deb.frrouting.org
+
+RPM Packages - https://rpm.frrouting.org
+
+Snaps - https://snapcraft.io/frr
+
+Docker - [quay.io/frrouting/frr:9.1.1](https://quay.io/repository/frrouting/frr/manifest/sha256:)
```
Comment (ton31337) [inline content/release/9.1.1.md:19]: https://quay.io/repository/frrouting/frr/manifest/sha256:0634cca8750c3ed2122a4bc79d65c932b091d1afba19dd617cd66a4adf923db3

File: content/release/9.0.3.md (hunk 10 above)
Comment (ton31337) [reply content/release/9.0.3.md:125]: Wrong still

File: content/release/9.1.1.md (hunk 12 above)
Comment (ton31337) [reply content/release/9.1.1.md:154]: Still wrong

**Review by ton31337 — COMMENTED** [review]
Missing some comments yet

**Pull request merged**